doc/release-notes.md

```diff
@@ -53,6 +116,62 @@ this includes watch-only addresses linked to viewing keys imported with
 `z_importviewingkey`, as well as addresses with spending keys (both generated
 with `z_getnewaddress` and imported with `z_importkey`).
 
+Better error messages for rejected transactions after network upgrades
+----------------------------------------------------------------------
+
+The Zcash network upgrade process includes several features designed to protect
+users. One of these is the "consensus branch ID", which prevents transactions
+created after a network upgrade has activated from being replayed on another
+chain (that might have occurred due to, for example, a
+[friendly fork](https://electriccoin.co/blog/future-friendly-fork/)). This is
+known as "two-way replay protection", and is a core requirement by
+[various](https://blog.bitgo.com/bitgos-approach-to-handling-a-hard-fork-71e572506d7d?gi=3b80c02e027e)
+[members](https://trezor.io/support/general/hard-forks/) of the cryptocurrency
+ecosystem for supporting "hard fork"-style changes like our network upgrades.
+
+One downside of the way replay protection is implemented in Zcash, is that there
```

Comment:
non-blocking reword: One downside of the implementation of replay protection in `zcashd` is the lack of a visible difference between transactions rejected for targeting a different branch and transactions rejected due to an invalid signature.